features/kinderen.feature

```diff
@@ -164,10 +175,7 @@ Functionaliteit: Kinderen van een persoon raadplegen
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 555550005                       |
       | fields              | burgerservicenummer,kinderen    |
-      Dan heeft de persoon met burgerservicenummer '555550005' een 'kind' met alleen de volgende gegevens
-      | naam              | waarde       |
-      | type              | KindOnbekend |
-      | indicatieOnbekend | true         |
+      Dan heeft de persoon met burgerservicenummer '555550005' een 'kind' met zonder gegevens
```

Comment:
in `... met zonder gegevens` kan de `met` weg

Reply:
Aangpast